rust/rama/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "server"
+version = "0.0.0"
+edition = "2021"
+publish = false
+
+[profile.release]
+opt-level = 3
+debug = false
+debug-assertions = false
+lto = true
+panic = "abort"
+incremental = false
+codegen-units = 1
+rpath = false
+strip = false
+
```

Comment:
```suggestion
```
to make sure that we compile the same way all rust frameworks, compilation options are now in `rust/Dockerfile` instead of each `Cargo.toml`